proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+
+### Formal Review (up to 5 iterations)
+
+While the majority of general technical commentary and feedback will occur in the proposals repo, it's hard to tell when a particular proposal is finalized. This what we call "Formal Review".  
+
+#### Formal Review 
+
+Formal Review of a proposal is done in iterations. These iterations take place in the HF Tech Track meetings and are usually monthly. However, they can last longer, in which case the author has more time to implement all the required changes. 
+
+The maximum number of iterations is five. At the fifth iteration, the HF Technical Track can only vote to Accept, Postpone, or Reject.
+
+The HF Technical Track decides the duration of the next iteration depending upon the feedback and complexity of the HIP. Consequently, authors have more time to prepare all the changes. If they finish their revision before the scheduled iteration, an HF Technical Track member will reschedule it for the next available meeting.
+
+During every iteration, the HF Technical Track reviews the changes (updated design document, progress with the implementation, etc) to the proposal. Based on the feedback, the HIP is either:
+
+1. **Accepted**, in which case the HF Technical Track has accepted the proposal, and it will be merged by the HF Technical Track members.
+2. **Rejected**, in which case the HIP is closed and no longer evaluated in the future.
+3. **Postponed**, in which case the HF Technical Track sets aside the HIP under some conditions. When those conditions are met, the HIP can be resubmitted.
+4. **Under revision**, in which case the author needs to continue the formal evaluation and address all the HF Technical Track feedback. Thus, the follow-up discussion is scheduled for the next iteration. 
+5. **Dormant**,  in which case no changes have been made to a HIP in two iterations, it’s marked as dormant and both the PR and issue are closed. Dormant HIPs can be reopened by any person, be it the same or different authors, at which point it will start from the formal evaluation phase.
+
+#### Involving Stakeholders
+
+At this point where a propsal is being reviewed, if there are relevant stakeholders from industry or in the community (e.g. library authors and maintainers who are affected), they should be solicited for comment and brought into the discussion. Remember, proposals may sound good, but it's best to iron out as many details as possible prior to their acceptance. This means that any hesitations stakeholders have with the project should be factored into the proposal's details. 
+
+#### Updating Proposal Statuses
+
+If the author of a particular proposal wants to update the status of their proposal, they should either comment on the issue and tag an HF Technical Track member with a note regarding what status they would like to update the proposal to. It is good to be redundant and raise the question in our other fora as well: Slack and Discourse. However, Github should be the first choice. 
```

Comment:
"tag an HFTT member"

Which one? I'm trying to think of myself as an author, and this instruction seems confusing, requiring me to look up the HFTT membership, track down their GitHub names, and then arbitrarily choose one, hoping not to offend anyone. We should either designate an individual (essentially: the HFTT secretary) or have some sort of group tag (does GitHub even have this?).

"GitHub should be the first choice": This, to me, implies that an author who dislikes GitHub may choose to just make a small comment on Slack. That comment might get missed, and then the author gets frustrated. I think we should clarify that all official requests for action on a proposal take place through GitHub.

Reply:
Github does have group tags (ala @haskellfoundation/tech-proposals), which requires adding HFTT members to a team on this repo. This is easy to do, and I would prefer this as opposed to adding a new role.